src/routes/PublicLayoutRouter/PublicLayoutRouter.js

```diff
@@ -133,7 +136,9 @@ export function GroupExplorerLayoutRouter () {
   return (
     <>
       <div styleName='center-column' id={CENTER_COLUMN_ID}>
-        <GroupExplorer />
+        <div>
```

Comment:
Curious why the extra `div` fixed things here? It's nitpick a bit, but not a fan of extra containers unless they're truly necessary, in most all cases I find that adjusting the flex box layout of the existing container is good enough?

Reply:
The whole complexity of the auth styling for this, with the nav div, the center div and the detail div, have not been ported over. This was an extra fix, beyond the scope of this ticket, just to have things looking vaguely correct.